modular_skyrat/code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -351,7 +351,7 @@
 /obj/item/clothing/suit/space/hardsuit/deathsquad/praetor
 	name = "Praetor Suit"
 	desc = "And those that tasted the bite of his sword named him... The Doom Slayer."
-	armor = list("melee" = 75, "bullet" = 55, "laser" = 55, "energy" = 45, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
+	armor = list("melee" = 80, "bullet" = 20, "laser" = 20, "energy" = 30, "bomb" = 70, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)
```

Comment:
Bullet, laser, and energy shouldn't be the same. If you make one high, you have to make the other low. This is basic balance. if it's the same thing, with melee being absolutely insane, then the ways to counter it are then nullified.

Reply:
20 is low. These are not good stats and bullets and laser can still easily counter the suit. Same for energy. Stop trying to move the goalpost.